src/goto-instrument/goto_program2code.cpp

```diff
@@ -1671,12 +1668,12 @@ void goto_program2codet::remove_const(typet &type)
     if(!const_removed.insert(identifier).second)
       return;
 
-    symbol_tablet::symbolst::iterator it=
-      symbol_table.symbols.find(identifier);
-    assert(it!=symbol_table.symbols.end());
-    assert(it->second.is_type);
+    symbolt &symbol=*symbol_table.get_writeable(identifier);
+    INVARIANT(
```

Comment:
the second assert (`..!=..end()`)has vanished, is this an invariant in `get_writeable` ?

Reply:
No, but it is of `operator*` against an `optionalt`